README.md

```diff
@@ -80,4 +80,58 @@ For openning pull requests you have to send PR's to [`develop`](https://github.c
         Accept: application/json; version=<your reuired version>
     
         example:
-            Accept: application/json; version=1.0.1
\ No newline at end of file
+            Accept: application/json; version=1.0.1
+
+
+
+## API Documentaion 
+Jobguy uses swagger to generate api docs
+
+you can see docs [here](https://api.jobguy.work/swagger/)
+
+if you head to api link, you'll see this page.
+
+this is the public API 
+
+![public api screen](./screenshots/public_api_screen.png)
+
+you can expand this parts and easily try them.
+
+jobguy has more than these public endpoints.
+
+if you want to access those endpoints, you need to authenticate.
+
+to do so, expand the **authnz** part,and you'll see something like this:
+
+![public api screen](./screenshots/auth_view_screen.png)
+
+### follow these steps for authentication 
+
+1. click on `/authnz/login_email/` endpoint
+   
+
+2. click on Example value box on right side, so the required body parameters will appear in data input, and fill it with your own data.
+   
+
+3. after you click on **Try it out!**, you'll get a response (like following image)
+
+![public api screen](./screenshots/token_included_response_screen.png)
+
+4. copy the token value, then click on `Authorize` button on top right of the screen, you'll see something like this:
+
+![public api screen](./screenshots/auth_pop_up_screen.png)
```

Comment:
Could you use an image that showing the start of your example value, it could help to understand it better. I mean your image show JWT your.jwt.token in value box. Your current image show the middle of token(missing the start with JWT).

Reply:
what do you mean ?
you  mean like this?:
![image](https://user-images.githubusercontent.com/84769510/125331553-615ed280-e35d-11eb-8eb1-25a77d17d0ce.png)

Reply:
![IMG_20210712_221230](https://user-images.githubusercontent.com/24853386/125332523-62dcca80-e35e-11eb-8890-5940f70e4640.jpg)

Reply:
![image](https://user-images.githubusercontent.com/84769510/125334001-1f835b80-e360-11eb-8fe6-7d8541d62e50.png)
is this ok?

Reply:
Yes